### PR TITLE
help(spec-support): re-organize the section

### PR DESCRIPTION
**Pull request by paulRbr**
This commit re-organizes the “specification support” section.

### CODE REVIEW

File: src/_help/specification-support/extensions.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+---
+
+- TOC
+{:toc}
+
+These are additional properties that are not specified by either the OpenAPI or the AsyncAPI specifications but are helpful for you to be able to customize your documentation content. All those properties start with the x- naming convention to be identified as “eXternal” from the OpenAPI or AsyncAPI specification.
+
+### Add topics to your documentation (`x-topics`)
+
+We created a custom property to enrich your documentation. This vendor-specific property helpsto add more context paragraphs in your generated documentation. Find out more in our [dedicated section](/help/enhance-documentation-content/topics/).
```
Comment (jayk-u) [inline src/_help/specification-support/extensions.md:12]:
```suggestion
We created a custom property to enrich your documentation. This vendor-specific property helps to add more context paragraphs in your generated documentation. Find out more in our [dedicated section](/help/enhance-documentation-content/topics/).
```

File: src/_help/specification-support/extensions.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+### Custom code sample examples (`x-codeSamples`)
+
+> This vendor extension is only available for an OpenAPI documents for now
+{: .info}
+
+This is another unsupported property by OpenAPI, we added a custom property so you can add your own code samples in one or more programming languages to your documentation. Find out more in our [dedicated section](/help/specification-support/doc-code-samples).
```
Comment (jayk-u) [inline src/_help/specification-support/extensions.md:19]:
```
This is another unsupported property by OpenAPI, we added a custom property so...
```

Not 100% sure of how this part starts. Like "another unsupported" feels a bit rough. WDYT?

**Review by jayk-u — COMMENTED** [review]
Great work Paulr :fire:

File: src/_help/specification-support/extensions.md (hunk 2 above)
Comment (paulRbr) [reply src/_help/specification-support/extensions.md:19]: you are right.

Comment (paulRbr) [inline src/_help/specification-support/extensions.md:19]:
```suggestion
This is another extra property not defined by the OpenAPI specification, we added a custom property so you can add your own code samples in one or more programming languages to your documentation. Find out more in our [dedicated section](/help/specification-support/doc-code-samples).
```

File: src/_help/quick-start/index.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ With your documentation ready, you could want to make it more customized. Bump.s
 
 You can setup [Custom Domains](/help/custom-domains), to make your doc available behind your own URL.
 Several branding options are available: from your logo to [Meta Images](/help/meta-images) (that helps sharing your doc in a fancy way).
-We also support [Documentation Topics](/help/doc-topics), that helps you adding rich (con)text to your API doc, sharing the informations your community needs.
+We also support [Documentation Topics](/help/enhance-documentation-content/topics/), that helps you adding rich (con)text to your API doc, sharing the informations your community needs.
```
Comment (ChristopheDujarric) [inline src/_help/quick-start/index.md:56]:
```suggestion
We also support [Documentation Topics](/help/enhance-documentation-content/topics/), which help you add rich (con)text to your API doc, sharing the information your community needs.
```

File: src/_help/specification-support/extensions.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+Title: Specification extensions
+---
+
+- TOC
+{:toc}
+
+These are additional properties that are not specified by either the OpenAPI or the AsyncAPI specifications but are helpful for you to be able to customize your documentation content. All those properties start with the x- naming convention to be identified as “eXternal” from the OpenAPI or AsyncAPI specification.
```
Comment (ChristopheDujarric) [inline src/_help/specification-support/extensions.md:8]:
```suggestion
These additional properties are not specified by the OpenAPI or the AsyncAPI specifications but can help you customize your documentation content. All those properties start with the x- naming convention to be identified as “eXternal” from the OpenAPI or AsyncAPI specification.
```

File: src/_help/specification-support/extensions.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+### Add topics to your documentation (`x-topics`)
+
+We created a custom property to enrich your documentation. This vendor-specific property helps to add more context paragraphs in your generated documentation. Find out more in our [dedicated section](/help/enhance-documentation-content/topics/).
```
Comment (ChristopheDujarric) [inline src/_help/specification-support/extensions.md:12]:
```suggestion
This vendor-specific property we created helps to add more context paragraphs in your generated documentation. Find out more in our [dedicated section](/help/enhance-documentation-content/topics/).
```

File: src/_help/specification-support/extensions.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+### Custom code sample examples (`x-codeSamples`)
+
+> This vendor extension is only available for an OpenAPI documents for now
```
Comment (ChristopheDujarric) [inline src/_help/specification-support/extensions.md:16]:
```suggestion
> This vendor extension is only available for OpenAPI documents for now
```

File: src/_help/specification-support/extensions.md (hunk 2 above)
Comment (ChristopheDujarric) [reply src/_help/specification-support/extensions.md:19]: "We added a custom property, not supported by OpenAPI, so you can add your own code samples in one or more programming languages to your documentation."

File: src/_help/specification-support/openapi-support.md [hunk 7]
```diff
@@ -5,6 +5,19 @@ title: OpenAPI support
 - TOC
 {:toc}
 
+We support all major versions from Swagger (OpenAPI v2), OpenAPI v3 and OpenAPI v3.1. This pages describes some specificities related to our support for this specification.
+
+## Minimal mandatory fields
+
+Bump.sh needs to receive at least these fields to generate a REST documentation
```
Comment (ChristopheDujarric) [inline src/_help/specification-support/openapi-support.md:12]:
```suggestion
Bump.sh needs to receive at least these fields to generate a REST documentation:
```

**Review by ChristopheDujarric — CHANGES_REQUESTED** [review]

File: src/_help/specification-support/openapi-support.md (hunk 7 above)
Comment (paulRbr) [reply src/_help/specification-support/openapi-support.md:12]: Not sure the punctuation helps that much as it's a short sentence without a full-top

**Pull request merged**